transformer_lens/utils.py

```diff
@@ -669,10 +669,10 @@ def transpose(tensor: Float[torch.Tensor, "... a b"]) -> Float[torch.Tensor, "..
 
 
 def composition_scores(
-    left: FactoredMatrix, right: FactoredMatrix, broadcast_dims=True
+    left: "FactoredMatrix", right: "FactoredMatrix", broadcast_dims=True
```

Comment:
```suggestion
    left: FactoredMatrix, right: FactoredMatrix, broadcast_dims=True
```
I would imagine the forward reference is not needed here therefore?

Reply:
I think it's actually needed to get beartype to not complain. I suspect it's because there's a circular dependency between utils.py and FactoredMatrix